get_client_token.py

```diff
@@ -0,0 +1,34 @@
+def get_client_token():
+    from requests.auth import HTTPBasicAuth
+    import requests
+
+    import os
+    from dotenv import load_dotenv
+
+    load_dotenv()
+    client_id = os.environ['CLIENT_ID']
+    client_secret = os.environ['CLIENT_SECRET']
+
+    # Perform HTTP Basic Authentication
+    auth = HTTPBasicAuth(client_id, client_secret)
+
+    # Set the grant type for client credentials
+    payload = {
+        'grant_type': 'client_credentials'
+    }
+
+    # Make the POST request
+    response = requests.post(
+        'https://allegro.pl/auth/oauth/token',
+        auth=auth,
+        data=payload
+    )
+
+    # Check if the request was successful
+    if response.ok:
+        # Parse the access token from the response
+        return response.json().get('access_token')
```

Comment:
nie wiem czy nie lepiej:
`response.json()['access_token']`
bo inaczej jak nie będzie pola to funkcja zwróci `None`